.changeset/red-avocados-talk.md

```diff
@@ -0,0 +1,14 @@
+---
+"@salt-ds/theme": minor
+---
+
+Added size tokens
+
+```diff
++ --salt-size-bar
++ --salt-size-selectable
++ --salt-size-icon
+```
+
+`--salt-size-base ` changed from a calculation on the basis unit to hardcoded values per density
```

Comment:
potential breaking change - unsure how to handle

Reply:
This wouldn't be considered a breaking change since the API is the same